dao/mongodb.rya/src/main/java/mvm/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -167,7 +164,7 @@ public static RyaDetails toRyaDetails(final DBObject mongoObj) throws MalformedR
         return RyaDetails.builder()
             .setRyaInstanceName(basicObj.getString(INSTANCE_KEY))
             .setRyaVersion(basicObj.getString(VERSION_KEY))
-            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(basicObj.getBoolean(ENTITY_DETAILS_KEY)))
+            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(false))//entity indexing is not supported
             .setGeoIndexDetails(new GeoIndexDetails(basicObj.getBoolean(GEO_DETAILS_KEY)))
             .setPCJIndexDetails(getPCJIndexDetails(basicObj))
```

Comment:
Since you removed the integration code, could we just assume it should be read from the db object here? This is going to be really confusing code otherwise.